src/components/TableBody.js

```diff
@@ -221,15 +236,21 @@ class TableBody extends React.Component {
 
             let isRowSelected = options.selectableRows !== 'none' ? this.isRowSelected(dataIndex) : false;
             let isRowSelectable = this.isRowSelectable(dataIndex);
+            let bodyClasses = (options.setRowProps ? options.setRowProps(row, dataIndex, rowIndex) : {});
 
             return (
               <React.Fragment key={rowIndex}>
                 <TableBodyRow
-                  {...(options.setRowProps ? options.setRowProps(row, dataIndex, rowIndex) : {})}
+                  {...bodyClasses}
                   options={options}
                   rowSelected={isRowSelected}
                   isRowSelectable={isRowSelectable}
                   onClick={this.handleRowClick.bind(null, row, { rowIndex, dataIndex })}
+                  className={classNames({
```

Comment:
What happens to the deprecated options?

Reply:
They will all still work. These classes are just to remove the extra line that occurs under the last cell when its stacked.